armada/tiller.py

```diff
@@ -99,6 +105,13 @@ def update_release(self, chart, dry_run, name, disable_hooks=False,
         else:
             values = Config(raw=values)
 
+        print name
+        for job in jobs:
+            # LOG.info("Deleting [%s] : %s", "job", items.name)
+            print name, job
+            body = client.V1DeleteOptions()
+            self.k8s.api_client.delete_namespaced_job(name, job, body)
```

Comment:
This also requires the issue of not updating if no update is necessary get solved

We only want to delete the jobs if

a) they havent disabled pre 
b) there is an actual update to perform 
today, if a chart exists and there are 0 diffs, we STILL push a new release.  That needs to be fixed
c) only if we detect diffs and an update is imminent, would the pre workflow be done